onnxruntime/core/providers/dml/DmlExecutionProvider/src/DmlGraphFusionHelper.cpp

```diff
@@ -288,12 +287,25 @@ namespace DmlGraphFusionHelper
         return initializerPartitionMap;
     }
 
+    enum DML_PREVIEW_OPERATOR_TYPE
+    {
+        DML_PREVIEW_OPERATOR_FIRST = 0xC0000000,
+    };
+
+    struct DML_CONSTANT_DATA_GRAPH_NODE_DESC_PREVIEW 
```

Comment:
What's our strategy to reflect this with new DML1.13 header for the DML_CONSTANT_DATA_GRAPH_NODE_DESC ?

Reply:
Yes, we should update this as per the new DirectML.h header.